content/en/docs/concepts/cluster-administration/system-traces.md

```diff
@@ -9,7 +9,8 @@ weight: 60
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.22, kube-apiserver" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.25, kubelet" state="alpha" >}}
```

Comment:
I'm afraid that this use isn't idiomatic. The version should always be just a version string.

Reply:
I wasn't sure how to list each separately, I know. I tried a few different ways. Do you have a suggestion? Or, should I only list the latest? or the earliest? There will be other components added here, too (scheduler next, maybe?)

Reply:
updated, added new feature state shortcode under kubelet traces